src/dpp/utility.cpp

```diff
@@ -299,7 +300,48 @@ namespace dpp {
 		uint32_t cmyk(int c, int m, int y, int k) {
 			return cmyk(c / 255.0, m / 255.0, y / 255.0, k / 255.0);
 		}
+
+		uint32_t hsl(int h, int s, int l) {
+			double hue = static_cast<double>(h) / 360.0;
+			double saturation = static_cast<double>(s) / 100.0;
+			double lightness = static_cast<double>(l) / 100.0;
+			return hsl(hue,saturation,lightness);
```

Comment:
[This does not follow the code convention](https://dpp.dev/coding-standards.html#:~:text=braces%20with%20statements!-,lists,-Lists%20should%20have). It needs to be `hsl(hue, saturation, lightness)`